backend/hitas/tests/factories/housing_company.py

```diff
@@ -22,6 +23,8 @@ class Meta:
     display_name = factory.Sequence(lambda n: f"Test Housing company {n:03}")
     official_name = factory.LazyAttribute(lambda self: f"As Oy {self.display_name}")
     state = fuzzy.FuzzyChoice(state[0] for state in HousingCompanyState.choices())
+    exclude_from_statistics = False
+    regulation_status = fuzzy.FuzzyChoice(state[0] for state in RegulationStatus.choices())
```

Comment:
I'd rather have the `regulation_status` default value be `RegulationStatus.REGULATED`, so it doesn't need to be specified for every case separately.
Then when we need a non-regulated housing company we can specify it.

Reply:
Sure, I'll change it